README.md

```diff
@@ -95,6 +97,65 @@ Data is exported in JSON in this format
 }]
 ```
 
+## CSV format
+
+### Deliveries
+CSV file with deliveries have following format:
+```csv
+delivery.id,itemGroupId,item.id,item.quantity,parcel.id,parcel.length,parcel.height,parcel.width,parcel.weight,parcel.trackingId,parcel.carrier,parcel.provider,parcel.providerTransaction,parcel.isReturn
```

Comment:
Add underscore to `itemGroupId` because it is an internal property + explain more this property so the user will know how to use it + comment it in the code